agent/tool-scripts/lockstat

```diff
@@ -107,19 +107,28 @@ done
 
 tool_dir="$dir/tools-$group"
 tool_output_dir=$tool_dir/$tool # all tools keep data in their tool specific dir
-tool_ouput_file=$tool_output_dir/$tool.txt
+tool_output_file=$tool_output_dir/$tool.txt
 
 case "$mode" in
 	install)
-        ;;
+	if [[ ! -s "/proc/lock_stat" ]]; then
+		echo "please boot the system with CONFIG_LOCK_STAT=y to collect statistics on locks"
```

Comment:
I don' t think you can *boot* a kernel with that option. You can only *build* a kernel with that option enabled (e.g. debug kernels should be built with that option, but "normal" kernels are not).